molo/profiles/templates/registration/login.html

```diff
@@ -32,7 +32,7 @@ <h4 class="heading heading--subtitle">
 		<span class="input-hint">({% trans "Eg.: 2086" %})</span>
 	</fieldset>
 	<input type="submit" value="{% trans "Log in" %}">
-	<input type="hidden" name="next" value="{% if request.GET.next %}{{ request.GET.next }}{% else %}{{request.site.root_page.url}}{% endif %}" />
+	<input type="hidden" name="next" value="{% url "molo.profiles:login_success" %}" />
```

Comment:
@Ntu2koTrevor we still need to keep the `{{ request.GET.next }}` if there is a next value